CMakeLists.txt

```diff
@@ -116,6 +116,7 @@ option(VELOX_ENABLE_PARQUET "Enable Parquet support" OFF)
 option(VELOX_ENABLE_ARROW "Enable Arrow support" OFF)
 option(VELOX_ENABLE_REMOTE_FUNCTIONS "Enable remote function support" OFF)
 option(VELOX_ENABLE_CCACHE "Use ccache if installed." ON)
+option(VELOX_ENABLE_JNI_BINDINGS "Enable jni bindings." ON)
```

Comment:
nit: we should probably leave this off by default, and let users explicitly turn it on when needed.